content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -133,507 +157,583 @@ types available by default:
 
-.. image:: argentina/argentina_edi_18.png
+.. image:: argentina/afip-auth-rejected.png
    :align: center
-
+   :alt: AFIP authorization Rejected.
 
 Special Use cases
 ~~~~~~~~~~~~~~~~~
+
 Untaxed Concepts
 ****************
-There are some transactions that include items that are not part of the VAT base amount, 
-this is commonly used in fuel and gasoline invoices. 
 
-The vendor bill will be registered using 1 item for each product that is part of the VAT
-base amount and an additional item to register the amount of  the Exempt concept:
+There are some transactions that include items that are not a part of the VAT base amount, such as
+fuel and gasoline invoices.
 
-.. image:: argentina/argentina_edi_19.png
+The vendor bill will be registered using one item for each product that is part of the VAT base
+amount, and an additional item to register the amount of the Exempt concept.
+
+.. image:: argentina/vat-exempt.png
    :align: center
+   :alt: VAT exempt.
 
 Perception Taxes
 ****************
-The vendor bill will be registered using 1 item for each product that is part of the 
-VAT base amount, the perception tax can be added in any of the product lines, as result 
-we will have one tax group for the VAT and one for the perception, the perception default 
-value is always 1.00. 
 
-.. image:: argentina/argentina_edi_20.png
+The vendor bill will be registered using one item for each product that is part of the VAT base
+amount, and the perception tax can be added in any of the product lines. As a result, there will be
+one tax group for the VAT and another for the perception. The perception default value is always
+:guilabel:`0.10`.
+
+.. image:: argentina/vat-perception.png
    :align: center
+   :alt: VAT perception.
+
+To edit the VAT perception and set the correct amount, you should use the :guilabel:`Pencil` icon
+that is the next to the :guilabel:`Perception` amount. After the VAT perception amount has been
+set, the invoice can then be validated.
 
-You should use the pencil that is the next to the Perception amount to edit it 
-and set the correct amount. 
+Then invoice can be validated.
 
-.. image:: argentina/argentina_edi_21.png
+.. image:: argentina/enter-perception-amount.png
    :align: center
-   
-After this is done the invoice can be validated. 
-   
-   
+   :alt: Enter the perception amount.
+
 Reports
 =======
 
-As part of the localization the next Financial reports were added:
+As part of the localization installation, Financial reporting for Argentina was added inside the
+:guilabel:`Accounting` dashboard. Access these reports by navigating to :menuselection:`Accounting
+--> Reporting --> Argentinian Reports`
 
-.. image:: argentina/argentina18.png
+.. image:: argentina/argentinian-reports.png
    :align: center
+   :alt: Argentinian reports.
 
 VAT Reports
 -----------
 
-Libro de IVA Ventas
-~~~~~~~~~~~~~~~~~~~
+Sales VAT book
+~~~~~~~~~~~~~~
+
```

Comment:
Below you can find the missing text:
_In this report, all the sales are recorded, which are taken as the basis for the accounting records to determine the VAT (Tax Debit)._
@samueljlieber